src/Parser/Core/Modules/Abilities.js

```diff
@@ -20,6 +20,7 @@ class Abilities extends Analyzer {
     OTHERS: 'Spell',
     UTILITY: 'Utility',
     HEALER_DAMAGING_SPELL: 'Damaging Spell',
+    HIDDEN: '',
```

Comment:
I'm a little uncomfortable with this using the empty string. Since by its nature it's not going to be seen anyway why not just make it 'Hidden'?